std-bits/google-api/src/main/java/org/enso/google/GoogleCredentialsProvider.java

```diff
@@ -0,0 +1,26 @@
+package org.enso.google;
+
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.api.gax.core.FixedCredentialsProvider;
+
+public class GoogleCredentialsProvider {
+  private final CredentialsProvider provider;
+
+  private GoogleCredentialsProvider(CredentialsProvider provider) {
+    this.provider = provider;
+  }
+
+  public static GoogleCredentialsProvider fromGoogleCredentials(
+      WrappedGoogleCredentials googleCredentials) {
+    return switch (googleCredentials) {
+      case WrappedGoogleCredentials.SecretCredentials ref -> null;
```

Comment:
I'd prefer to throw an exception i.e. `IllegalArgumentException` which would be clearer than an NPE?

Reply:
I believe that is handled correctly in the implementation